xformers/ops/swiglu.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+
+class SwiGLUFFN_Reference(nn.Module):
```

Comment:
I wonder why provide this implementation? Can't we just test the numerics with a functional API?